apps/bajour/next.config.js

```diff
@@ -43,7 +43,57 @@ const nextConfig = {
       }
     }
   },
-  transpilePackages: ['@wepublish/ui', '@wepublish/website', 'react-tweet']
+  transpilePackages: ['@wepublish/ui', '@wepublish/website', 'react-tweet'],
+  async redirects() {
+    return [
+      {
+        source: '/home',
```

Comment:
Because the home page has a slug of /home, we probably should change that in the future